src/hardware/mixer.cpp

```diff
@@ -121,6 +124,16 @@ MixerChannel * MIXER_AddChannel(MIXER_Handler handler, Bitu freq, const char * n
 	chan->MapChannels(0, 1);
 	chan->Enable(false);
 	mixer.channels=chan;
+
+	const auto mix_rate = mixer.freq;
+	const auto chan_rate = chan->GetSampleRate();
+	if (chan_rate == mix_rate)
+		LOG_MSG("MIXER: %s channel at %u Hz playing without resampling",
+		        name, chan_rate);
+	else
+		LOG_MSG("MIXER: %s channel at %u Hz will be %s to the mixer's rate",
```

Comment:
I believe this will print "… downsampled to to ..." ;)

Reply:
Great catch @dreamer ! will update.